doc/cli/npm-completion.md

```diff
@@ -3,7 +3,10 @@ npm-completion(1) -- Tab Completion for npm
 
 ## SYNOPSIS
 
-    . <(npm completion)
+    npm completion >> ~/.bashrc
+    npm completion >> ~/.zshrc
+
+    source <(npm completion)
```

Comment:
The way this is written is a little confusing. Maybe move the `npm completion >> ~ /.{z,ba}shrc` stuff into a section down below?